src/dotnet/commands/dotnet-publish/Program.cs

```diff
@@ -97,7 +97,7 @@ public static PublishCommand FromArgs(string[] args, string msbuildPath = null)
 
                 if (!string.IsNullOrEmpty(filterProjOption.Value()))
                 {
-                    msbuildArgs.Add($"/p:FilterProjFile={filterProjOption.Value()}");
+                    msbuildArgs.Add($"/p:FilterProjectFiles={filterProjOption.Value()}");
```

Comment:
@livarcocc @eerhardt this is the minimum change, the correct change will add multiple files.
Is it fine if i add it in a subsequent PR ?